frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/EditTab.tsx

```diff
@@ -44,14 +48,9 @@ export function EditTab({ component, handleComponentChange }: EditTabProps): Rea
     }
   }, [component, previousComponent]);
 
-  switch (status) {
+  switch (mergeQueryStatuses(statusOptionListIds, statusTextResources)) {
```

Comment:
What do you think about moving the function call into a variable to make it more readable? 😄 
```suggestion
   const mergedQueryStatues: boolean = mergeQueryStatuses(statusOptionListIds, statusTextResources)

  switch (mergedQueryStatues) {
```